src/yew_component.rs

```diff
@@ -73,27 +73,27 @@ where
     type Message = ();
     type Properties = AnsiProps<Text>;
 
-    fn create(props: Self::Properties, _link: ComponentLink<Self>) -> Self {
+    fn create(ctx: &Context<Self>) -> Self {
         let mut instance = Self {
-            props,
+            props: ctx.props().clone(),
             segments: Vec::new(),
             _builder: PhantomData::default(),
         };
         instance.update_segments();
         instance
     }
 
-    fn update(&mut self, _msg: Self::Message) -> ShouldRender {
-        false
+    fn update(&mut self, _ctx: &Context<Self>, _msg: Self::Message) -> bool {
+        true
     }
 
-    fn change(&mut self, props: Self::Properties) -> ShouldRender {
-        let update_segments = self.props.text != props.text;
+    fn changed(&mut self, ctx: &Context<Self>, _old: &Self::Properties) -> bool {
```

Comment:
Really, this logic can be simplified: since yew now provides new and old properties to compare, we can just use this rather than storing them in `self`.